subsys/bluetooth/host/hci_core.c

```diff
@@ -21,12 +21,14 @@
 
 #define BT_DBG_ENABLED IS_ENABLED(CONFIG_BLUETOOTH_DEBUG_HCI_CORE)
 #include <bluetooth/log.h>
+#include <common/log.h>
 #include <bluetooth/bluetooth.h>
 #include <bluetooth/conn.h>
 #include <bluetooth/l2cap.h>
 #include <bluetooth/hci.h>
 #include <bluetooth/hci_driver.h>
 #include <bluetooth/storage.h>
+#include <common/rpa.h>
```

Comment:
@jhedberg, I got inspired from drivers/bluetooth/hci/h4.c  which uses "../util.h", and it conveys clearly what-is-from-where.